### PR TITLE
Mss pitch in wizard and congrats page [MAILPOET-2142]

### PR DESCRIPTION
**Pull request by pavel-mailpoet**
[MAILPOET-2142]

[MAILPOET-2142]: https://mailpoet.atlassian.net/browse/MAILPOET-2142

### CODE REVIEW

File: lib/AdminPages/Pages/Newsletters.php [hunk 1]
```diff
@@ -72,6 +78,7 @@ function render() {
     $data['segments'] = $segments;
     $data['settings'] = $this->settings->getAll();
     $data['mss_active'] = Bridge::isMPSendingServiceEnabled();
+    $data['has_premium_key'] = Bridge::isMSSKeySpecified();
```
Comment (NeosinneR) [inline lib/AdminPages/Pages/Newsletters.php]: This naming is a bit misleading - is this about Premium or MSS features?

**Comment by codemonkey-jack** [comment]
This fails QA. The image doesn't load on the new pitch page. 

image

The image link is: https://ps.w.org/mailpoet/assets/welcome-wizard/illu-pitch-mss.20190912.png

**Comment by codemonkey-jack** [comment]
@pavel-mailpoet Just so you know this is pending Quentin's response on the Jira task: https://mailpoet.atlassian.net/secure/RapidBoard.jspa?rapidView=3&projectKey=MAILPOET&modal=detail&selectedIssue=MAILPOET-2142

Once he's responded we can resolve the conflict and merge.

**Comment by codemonkey-jack** [comment]
@pavel-mailpoet This is now good to go. If you can resolve the conflicts and then I'll approve and merge.

**Pull request merged**